app/utils/FeesLookup.js

```diff
@@ -29,7 +29,7 @@ const createCall = async (applicationId, data, authToken) => {
         status: 'success',
         total: 0
     };
-    logInfo(`Sending fee request to api with the following payload: ${JSON.stringify(data)}`, applicationId);
+    logInfo(`Sending fee request to api with the following payload: ${JSON.stringify(data)} for auth=` + authToken, applicationId);
```

Comment:
can you revert this change please - dont know how it got in

Reply:
Looks like you committed it yourself :)

Reply:
See the commit at the top: https://github.com/hmcts/probate-caveats-frontend/pull/285/commits

Reply:
yes I know but I cant remove it

Reply:
whats all the autocomplete off changes?

Reply:
Ok, I'll remove it for you

Reply:
So the a11y tests don't fail.

Reply:
That change on FeesLookup.js is now gone.